demos/angular/ckeditor5/README.md

```diff
@@ -66,7 +66,7 @@ You will also see any lint errors in the console.
     export class AppComponent {
     
         // Define the initial content. It can be empty or undefined.
-        public content: string = '<p class="text"> Double click on the following formula to edit it.</p><p style="text-align:center;"><math><mi>z</mi><mo>=</mo><mfrac><mrow><mo>-</mo><mi>b</mi><mo>&PlusMinus;</mo><msqrt><msup><mi>b</mi><mn>3</mn></msup><mo>-</mo><mn>4</mn><mi>a</mi><mi>c</mi></msqrt></mrow><mrow><mn>2</mn><mi>a</mi></mrow></mfrac></math></p>';
+        public content: string = '<p class="text"> Double click on the following formula to edit it.</p><p style="text-align: center;"><math><mi>z</mi><mo>=</mo><mfrac><mrow><mo>-</mo><mi>b</mi><mo>&PlusMinus;</mo><msqrt><msup><mi>b</mi><mn>3</mn></msup><mo>-</mo><mn>4</mn><mi>a</mi><mi>c</mi></msqrt></mrow><mrow><mn>2</mn><mi>a</mi></mrow></mfrac></math></p>';
```

Comment:
I'm guessing these changes are due to the linter, is that right?

Reply:
Yes, it's not a broken link but the .md linting recommends to change that. Since it's only in a very few documents and just one or two lines at most, i thought it would be OK to add them here instead of creating a new branch with that purpose.

Reply:
I completely agree.